docs/source/user_guide/cuspatial_api_examples.ipynb

```diff
@@ -166,8 +166,9 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    # Create Comparator cuSpatial GeoSeries from a comparator point
</pre><p>I would add to comment: <code># Note this will broadcast the comparator point to all rows of the dataframe, doubling its size in memory</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>